### PR TITLE
[AC-43] - Admin Console HealthCheck service implementation

### PR DESCRIPTION
**Pull request by DavidJGapCR**

### CODE REVIEW

File: Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/ApplicationTests.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        _adminApiCaller = new AdminApiCallerFake();
+        _odsApiCaller = new OdsApiCallerFake();
+
+        _application = new Application(null, _adminApiCaller, _odsApiCaller);
+    }
+
+    [Test]
+    public void GivenACallToOdsApi_ShouldReturnHealthCheckData()
+    {
+        "123".ShouldBe("123");
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/ApplicationTests.cs:33]: Is this the correct implementation?

File: Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/Features/AdminApi/AdminApiCallerTests.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    [SetUp]
+    public void SetUp()
+    {
+        _logger = A.Fake<ILogger<InstanceValidatorTests>>();
+        _fakeAdminApiClient = new AdminApiClientFake();
+        _adminApiCaller = new AdminApiCaller(_logger, _fakeAdminApiClient, Testing.GetAdminApiSettings());
+    }
+
+    [Test]
+    public async Task GivenACallToAdminApi_ShouldReturnInstances()
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/Features/AdminApi/AdminApiCallerTests.cs:31]: The test is not passing. I'm getting the following error:

```
 Shouldly.ShouldAssertException : instances.Count()
      should be
  2
      but was
  0

Stack Trace: 
  AdminApiCallerTests.GivenACallToAdminApi_ShouldReturnInstances() line 34
  ExceptionDispatchInfo.Throw()
  TaskAwaiter.ThrowForNonSuccess(Task task)
  TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task, ConfigureAwaitOptions options)
  TaskAwaiter`1.GetResult()
  GenericAdapter`1.GetResult()
  AsyncToSyncAdapter.Await[TResult](Func`1 invoke)
  AsyncToSyncAdapter.Await(Func`1 invoke)
  TestMethodCommand.RunTestMethod(TestExecutionContext context)
  TestMethodCommand.Execute(TestExecutionContext context)
  <>c__DisplayClass1_0.<Execute>b__0()
  DelegatingTestCommand.RunTestMethodInThreadAbortSafeZone(TestExecutionContext context, Action action)
```

File: Docker/healthcheck.Dockerfile [hunk 3]
```diff
@@ -3,22 +3,20 @@
...
-# EdFi.AdminConsoleHealthCheckService.Dockerfile
-
 # Image based on .NET SDK to compile and publish the application
 FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
 WORKDIR /source
 
-# Copy EdFi.AdminConsoleHealthCheckService.csproj file
-COPY ../Application/EdFi.AdminConsoleServices.sln .
-COPY ../Application/EdFi.AdminConsoleHealthCheckService/EdFi.AdminConsoleHealthCheckService.csproj ./EdFi.AdminConsoleHealthCheckService/
+# Copy EdFi.Ods.AdminApi.AdminConsole.HealthCheckService.csproj file
```
Comment (jagudelo-gap) [inline Docker/healthcheck.Dockerfile:10]: Remember to update the name

File: Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/ApplicationTests.cs (hunk 1 above)
Comment (DavidJGapCR) [reply Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/ApplicationTests.cs:33]: Removed

File: Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/Features/AdminApi/AdminApiCallerTests.cs (hunk 2 above)
Comment (DavidJGapCR) [reply Application/EdFi.Ods.AdminApi.HealthCheckService.UnitTests/Features/AdminApi/AdminApiCallerTests.cs:31]: Fixed

**Review by jagudelo-gap — APPROVED** [review]

**Pull request merged**